runtime/lua/vim/treesitter.lua

```diff
@@ -45,6 +44,22 @@ setmetatable(M, {
    end
  })
 
+function M.require_language(lang, path)
+  if vim._ts_has_language(lang) then
+    return true
+  end
+  if path == nil then
+    local fname = 'treesitter/parser/' .. lang .. '.*'
```

Comment:
we should just use `parser/` as the runtime directory, don't need to qualify it with `treesitter/`.

Reply:
done